docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -56,18 +56,20 @@ The deployer account is the account that will deploy your contracts. Additionall
 
 You can generate a random account / private key or add your crypto wallet's private key.
 
-To create a random account and add the `DEPLOYER_PRIVATE_KEY` to the `.env` file, run:
+To create a random account and add the `DEPLOYER_PRIVATE_KEY_ENCRYPTED` to the `.env` file, run:
```

Comment:
I think we can remove this line completely since user doesn't / shouldn't manually update the `DEPLOYER_PRIVATE_KEY_ENCRYPTED` in `.env` and it will only be touched by CLI